@xen-orchestra/babel-config/index.js

```diff
@@ -61,7 +67,13 @@ module.exports = function(pkg, plugins, presets) {
   return {
     comments: !__PROD__,
     ignore: __TEST__ ? undefined : [/\.spec\.js$/],
-    plugins: Object.keys(plugins).map(plugin => [plugin, plugins[plugin]]),
+    plugins: Object.keys(plugins)
+      .map(plugin => [plugin, plugins[plugin]])
+      .sort(([a], [b]) => {
+        const oA = pluginsOrder.indexOf(a)
+        const oB = pluginsOrder.indexOf(b)
+        return oA !== -1 && oB !== -1 ? oA - oB : a < b ? -1 : 1
+      }),
```

Comment:
What are these changes?

Reply:
I wanted to use the `plugin-proposal-class-properties`. So i copied/pasted  the needed change from the PR https://github.com/vatesfr/xen-orchestra/pull/4061/files#diff-42a8658c151916b2327b79533be5eaa9R70.